integration-testing/rnode_testing/util.py

```diff
@@ -16,12 +19,14 @@ def log_box(log_function, title="", char="*", length=150):
         log_function(char * length)
 
 
-def make_tempfile(prefix, content):
+def make_tempfile(prefix: str, content: str) -> str:
     fd, path = tempfile.mkstemp(dir="/tmp", prefix=prefix)
 
     with os.fdopen(fd, 'w') as tmp:
         tmp.write(content)
 
+    return path
```

Comment:
The mypy test did find some error. This function used to return nothing which the [rnode code](https://github.com/rchain/rchain/pull/1929/files#diff-154a13f8a2df4a0e8a88d1db476928aeR293) should expect a str path to return.

Reply:
The question is why did the tests pass.  The return value must have been unused.